### DISCUSSION TITLE
cannot login after Keycloak install

### DISCUSSION THREAD

**Discussion by ordor2k** [tteck/Proxmox#3837] [Help Needed] [answered]
I tried with the instructions in this page:
https://github.com/tteck/Proxmox/discussions/2922
but I still got error "Invalid username or password."

**Comment by tteck** [comment, marked as answer]
Try
```
systemctl stop keycloak.service
cd /opt/keycloak
bin/kc.sh bootstrap-admin user --bootstrap-admin-username temp-admin --bootstrap-admin-password changeme
systemctl start keycloak.service
```

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/2922 updated.

**Reply by ordor2k** [reply]
you are the best!!!

**Reply by K3CAN** [reply]
@ordor2k  Did that work for you? 
When I run `bin/kc.sh bootstrap-admin user --bootstrap-admin-username temp-admin --bootstrap-admin-password changeme` I get an error: 
`Unknown option: 'bootstrap-admin'`

**Reply by ordor2k** [reply]
It did

**Comment by EndreEndi** [comment]
Hey, sorry to get this here, but anyone knows how to update the container to latest version?

**Reply by tteck** [reply]
@elad-bar submitted a PR for this https://github.com/tteck/Proxmox/pull/3834 You should be able to type `update` in the LXC console.
